packages/web-vite/src/pages/ReaderPage.tsx

```
@@ -134,6 +268,32 @@ const ReaderPage: React.FC = () => {
+                className="reader-label-chip"
+                style={{ backgroundColor: label.color }}
+              >
+                <svg
+                  className="label-chip-icon"
+                  width="14"
+                  height="14"
+                  viewBox="0 0 24 24"
+                  fill="white"
+                  stroke="none"
+                >
+                  <circle cx="12" cy="12" r="12" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)